lib/Driver/Compilation.cpp

```diff
@@ -266,6 +266,18 @@ namespace driver {
         parseable_output::emitBeganMessage(llvm::errs(), *BeganCmd, Pid);
     }
 
+    void
+    dependencyLoadFailed(StringRef DependenciesFile, bool Warn=true) {
+      if (Warn)
```

Comment:
I'm on the fence about this. On the one hand, we might get more feedback from users, but on the other they may end up with a warning they can't silence. (Neither Xcode nor SwiftPM expose a switch to turn off incremental builds.)